charades-ui/src/components/auth.tsx

```diff
@@ -0,0 +1,5 @@
+export default function Auth() {
+  console.log('Authorized');
+
+  return null;
+}
```

Comment:
Do we need authorization in our app? If yes, shouldn't this component wrap the all app?